boards/nxp/imx93_evk/imx93_evk_mimx9352_m33_ddr.dts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_imx93_m33.dtsi>
```

Comment:
Why not just include `imx93_evk_mimx9352_m33.dts` here?

Reply:
I followed the structure from imx8mp m7 ddr support: https://github.com/zephyrproject-rtos/zephyr/blob/a2ac676a106a757fe4cb2664bab35ea985da95f9/boards/nxp/imx8mp_evk/imx8mp_evk_mimx8ml8_m7_ddr.dts#L9

Should I use the structure from the imx95_evk?

Reply:
sure, if it makes more sense